docs/website/docs/reference/optimization-options.md

```diff
@@ -0,0 +1,54 @@
+# Optimization Options
+
+This page documents various supported flags for optimizing IREE programs. Each
+is presented with its English name, flag to enable/disable and default state.
```

Comment:
Oxford comma please ;)
```suggestion
This page documents various supported flags for optimizing IREE programs. Each
is presented with its English name, flag to enable/disable, and default state.
```

https://developers.google.com/style/commas#serial-commas

Reply:
I am saddened that guide exists.

Reply:
I've found it pretty useful actually, I guess I'm a technical writing nerd or something now 🤔